jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -1,63 +1,10 @@
 @Library('cilium') _
 
 pipeline {
-    agent {
-        label 'baremetal'
-    }
+    agent any
```

Comment:
we currently have 1 executor on master, this would block all other jobs from being scheduled, wouldn't it? I think it's safe to increase the executor count on master, but le'ts make this build schedule on master then and not block a bare metal node if it's not needed.

Reply:
Good thinking. I actually don't know if executor is blocked while doing nothing but waiting from sub-jobs, and the documentation is ambiguous about this.

This is easily tested though. If it is, we may want to try using `agent none` or something else. I will check :)

Reply:
With `label 'master'`, executor does get locked up and does not allow multiple pipelines to run concurrently.

However it looks like `agent none` is sufficient for `build` steps to be ran immediately even if the stages themselves do not have agents of their own. This allows multiple pipelines to run concurrently.